R/iSEEnrich.R

```diff
@@ -50,6 +67,10 @@ iSEEnrich <- function(sce,
   stopifnot(isScalarCharacter(organism))
 
   stopifnot(isScalarCharacter(gene_identifier))
+  
+  stopifnot(isScalarCharacter(clusters))
+  
+  stopifnot(isScalarCharacter(groups))
```

Comment:
Probably we could use some extra tests up ahead here, namely:

* that `clusters` and `groups` do exist in the colData, if specified by the user
* that the colData slot has some content in it - i know, it sounds very odd to have it empty, but sometimes something can go deeply wrong and colData has columns but e.g. no rows
* similar flavour: we should check that the desired redDim slot exists (the check is on redDimNames or something similar, IIRC)

Reply:
suggestions addressed with commit [337519d](https://github.com/NajlaAbassi/iSEEfier/pull/10/commits/337519dd92143bb5f1362e6e1bfcee49152ffd8b)